core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineMysql.java

```diff
@@ -206,10 +211,10 @@ public String getDataTypeForEngine(DataType scalarDbDataType) {
       case DATE:
         return "DATE";
       case TIME:
-        return "TIME";
+        return "TIME(6)";
```

Comment:
In this notation `TIME(N)`, N is the number of digits for the fractional seconds. Since the ScalarDB TIME type has a precision of 1 microsecond, there is a 6 digits precision. 
Meanwhile, the TIMESTAMP and TIMESTAMPTZ have a precision of 1 millisecond, so they have 3 digits for the fractional seconds.
The precision for these 3 types is the best common precision for all storage.

Reply:
I see. I had the wrong idea the ScalarDB TIME type has a precision of 1 millisecond. Thanks for the explanation!